### PR TITLE
netsniff-ng: 0.6.2 -> 0.6.3

### PR DESCRIPTION
**Pull request by dhess**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by adisbladis** [comment]
@GrahamcOfBorg build netsniff-ng

File: pkgs/tools/networking/netsniff-ng/default.nix [hunk 1]
```diff
@@ -20,6 +20,9 @@ stdenv.mkDerivation rec {
     libnetfilter_conntrack libpcap libsodium liburcu ncurses perl
     pkgconfig zlib ];
 
+  # trafgen can call out to cpp to process config files.
+  propagatedBuildInputs = [ stdenv.cc ];
```
Comment (adisbladis) [inline pkgs/tools/networking/netsniff-ng/default.nix]: Just putting `stdenv.cc` in `propagatedBuildInputs` won't make it available at runtime. You need to either patch the source to use nix-store path or use `wrapProgram`.

**Comment by dhess** [comment]
Fixed the `wrapProgram` issue.

**Comment by adisbladis** [comment]
Pushed in https://github.com/NixOS/nixpkgs/commit/fa3d1c5dfbc48b822661c67b69508765b51003fe

**Pull request closed without merging**